mne/utils/config.py

```diff
@@ -406,7 +406,11 @@ def _get_stim_channel(stim_channel, info, raise_error=True):
         stim_channel = [info['ch_names'][ch_] for ch_ in stim_channel]
     elif raise_error:
         raise ValueError("No stim channels found. Consider specifying them "
-                         "manually using the 'stim_channel' parameter.")
+                         "manually using the 'stim_channel' parameter. In "
+                         "many file formats, discrete markers are now "
+                         "available in raw.annotations. Consider using "
+                         "mne.events_from_annotations to convert these to "
+                         "events.")
```

Comment:
I would check that some annotations are present before tell people to use events_from_annotations

Reply:
Alright. I even added a test 😄.